packages/renderer/index.d.ts

```diff
@@ -487,7 +487,7 @@ declare namespace ReactPDF {
    * @see https://react-pdf.org/advanced#on-the-fly-rendering
    * @platform web
    */
-  class PDFDownloadLink extends React.Component<PDFDownloadLinkProps> {}
+  class PDFDownloadLink extends React.Component<PDFDownloadLinkProps> { }
```

Comment:
Can we revert these changes? Was good as it was before too and this is unrelated to PR